src/Button/Button.js

```diff
@@ -0,0 +1,102 @@
+import React, {Component, PropTypes} from 'react';
+import propTypes from './buttonPropTypes';
```

Comment:
It does yes, the modular approach was mostly an artefact of the merge process. No problem to pull it back rather than fighting `react-docgen`.